### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jhagberg**
Issue templates for bugs

### CODE REVIEW

**Review by HenrikeW — APPROVED** [review]
Looks good but I think it would be interesting to hear feedback from the test users on this, to see if it's understandable for non-tech-savvy people who are not used to report bugs. Some thoughts:
- Should we give them a hint how to find out what version their browser is? I just googled "What version is my firefox" to find out, but I'm not sure if everyone would do that. We could either tell them to google or describe where to find the information in the most common browsers. (In firefox and chrome it's in Help --> About <browser name>
- "Ytterligare sammanhang" sounds a bit weird to me. Is that auto-translated? Or is it just me, non-native Swedish speaker? I would rather phrase it "Ytterligare information". "Lägg till annan information, som du tror kan vara relevant för problemet, här", or something like that. 

I still approve, as these might be non-issues.

**Comment by jhagberg** [comment]
Yes it's just a first auto-translation you suggestions is much better I add that. We can also wait a bit with this merge

File: .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Skärmbilder**
+Om tillämpligt, lägg till skärmdumpar för att förklara ditt problem.
+
+**Skrivbord (vänligen fyll i följande information):**
+- OS: [t.ex. iOS, Windows]
+- Webbläsare [t.ex. Chrome safari]
+- Version [t.ex. 22]  [Tryck här för instruktioner](https://www.computerhope.com/issues/ch001329.htm)
+
+**Smartphone (fyll i följande information):**
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md]:
```suggestion
**Mobil enhet (fyll i följande information):**
```

File: .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- Enhet: [t.ex. iPhone 6]
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md]:
```suggestion
- Modell: [t.ex. iPhone 6]
```

File: .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+3. Scrolla ned till "..."
+4. Se fel
+
+**Förväntat beteende**
+En tydlig och kortfattad beskrivning av vad du förväntade dig att hända.
+
+**Skärmbilder**
+Om tillämpligt, lägg till skärmdumpar för att förklara ditt problem.
+
+**Skrivbord (vänligen fyll i följande information):**
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md]:
```suggestion
**Dator (vänligen fyll i följande information):**
```

File: .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+---
+
+**Beskriv felet**
+En tydlig och kortfattad beskrivning av vad felet är.
+
+**Hur man reproducerar**
+Steg för att reproducera beteendet/felet:
+1. Gå till "..."
+2. Klicka på '...'
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md]: Inconsistent quotes.

```suggestion
2. Klicka på "..."
```

File: .github/ISSUE_TEMPLATE/bug-report-en.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+- OS: [e.g. iOS]
+- Browser [e.g. chrome, safari]
+- Version [e.g. 22] [Link to instuctions](https://www.computerhope.com/issues/ch001329.htm)
+
+**Smartphone (please complete the following information):**
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/bug-report-en.md]: "Smartphone" is very American and not quite what we want to know.  We want to distinguish a desktop/laptop from a mobile device with a much smaller screen (mobile phone or tablet).

```suggestion
**Mobile device (please complete the following information):**
```

File: .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+- Version [t.ex. 22]  [Tryck här för instruktioner](https://www.computerhope.com/issues/ch001329.htm)
+
+**Smartphone (fyll i följande information):**
+- Enhet: [t.ex. iPhone 6]
+- OS: [t.ex. iOS8.1]
+- Webbläsare [t.ex. aktiewebbläsare, safari]
+- Version [t.ex. 22]
+
+**Ytterligare information**
+Lägg till någon annan inoformation om problemet här.
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md]:
```suggestion
Lägg till någon annan information om problemet här.
```

File: .github/ISSUE_TEMPLATE/bug-report-en.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report EN
+about: Create a report to help us improve
+title: ''
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/bug-report-en.md]: Consistent quoting and use of ellipses.

```suggestion
title: "..."
```

File: .github/ISSUE_TEMPLATE/bug-report-en.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/bug-report-en.md]:
```suggestion
1. Go to "..."
2. Click on "..."
3. Scroll down to "..."
```

File: .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
+---
+name: Rapportera fel på Svenska
+about: Skapa en felrapport för att hjälpa oss
+title: ''
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md]:
```suggestion
title: "..."
```

File: .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+**Förväntat beteende**
+En tydlig och kortfattad beskrivning av vad du förväntade dig att hända.
+
+**Skärmbilder**
+Om tillämpligt, lägg till skärmdumpar för att förklara ditt problem.
+
+**Skrivbord (vänligen fyll i följande information):**
+- OS: [t.ex. iOS, Windows]
+- Webbläsare [t.ex. Chrome safari]
+- Version [t.ex. 22]  [Tryck här för instruktioner](https://www.computerhope.com/issues/ch001329.htm)
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md]:
```suggestion
- Webbläsare: [t.ex. Chrome, Firefox, Safari]
- Version: [t.ex. 22]  [Länk till instruktioner](https://www.computerhope.com/issues/ch001329.htm)
```

File: .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+**Smartphone (fyll i följande information):**
+- Enhet: [t.ex. iPhone 6]
+- OS: [t.ex. iOS8.1]
+- Webbläsare [t.ex. aktiewebbläsare, safari]
+- Version [t.ex. 22]
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/rapportera-fel-p--svenska.md]: "Aktiewebbläsare"!

```suggestion
- Webbläsare: [t.ex. Chrome, Firefox, Safari]
- Version: [t.ex. 22]
```

File: .github/ISSUE_TEMPLATE/bug-report-en.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+**Desktop (please complete the following information):**
+- OS: [e.g. iOS]
+- Browser [e.g. chrome, safari]
+- Version [e.g. 22] [Link to instuctions](https://www.computerhope.com/issues/ch001329.htm)
+
+**Smartphone (please complete the following information):**
+- Device: [e.g. iPhone6]
+- OS: [e.g. iOS8.1]
+- Browser [e.g. stock browser, safari]
+- Version [e.g. 22]
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/bug-report-en.md]: Several of these suggestions here and further down:
1. Missing `:`.
2. Use the same list of main browsers.

```suggestion
- Browser: [e.g. Chrome, Firefox, Safari]
- Version: [e.g. 22]
```

File: .github/ISSUE_TEMPLATE/bug-report-en.md [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+- OS: [e.g. iOS]
+- Browser [e.g. chrome, safari]
+- Version [e.g. 22] [Link to instuctions](https://www.computerhope.com/issues/ch001329.htm)
```
Comment (andkaha) [inline .github/ISSUE_TEMPLATE/bug-report-en.md]:
```suggestion
- Browser: [e.g. Chrome, Firefox Safari]
- Version: [e.g. 22] [Link to instuctions](https://www.computerhope.com/issues/ch001329.htm)
```

**Review by andkaha — CHANGES_REQUESTED** [review]
Initial review of actual text. The text requires changes. I have not thought about whether something is missing but will go through it again soon.

**Comment by jhagberg** [comment]
Fixed suggestions changed a typo

**Review by andkaha — APPROVED** [review]
Looks ok.

**Pull request merged**